docker-compose.yml

```diff
@@ -60,12 +62,18 @@ services:
   # you are using an external postgres instance
   # This will be exposed at port 5432
   hoppscotch-db:
-    image: postgres
+    image: postgres:15
     ports:
       - "5432:5432"
+    user: postgres
     environment:
       # NOTE: Please UPDATE THIS PASSWORD!
       POSTGRES_PASSWORD: testpass
       POSTGRES_DB: hoppscotch
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready" ]
```

Comment:
@NicklasWallgren we could use 
`test: ["CMD-SHELL", "pg_isready -U $${POSTGRES_USER} -d $${POSTGRES_DB}"]`

Reply:
But why? As stated by the documentation "It is not necessary to supply correct user name, password, or database name values to obtain the server status;"

The `pg_isready` will return 0 either way, if the database is accepting a connection.

```bash
postgres@baf25b68a6a1:/$ pg_isready -U wrong_user -d wrong_password
/var/run/postgresql:5432 - accepting connections
postgres@baf25b68a6a1:/$ echo $?
0
```